edc-tests/e2e-tests/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ dependencies {
     testImplementation(libs.edc.api.contractnegotiation)
     testImplementation(libs.edc.api.transferprocess)
     testImplementation(libs.edc.spi.dataplane.selector)
-
+    testImplementation(project(":edc-extensions:control-plane-adapter-api"))
```

Comment:
micro-nit: put project dependencies first. And since you're touching this file: could you move the mockwebserver dependency to the version catalog?